game/scripts/npc/abilities/pugna_life_drain.txt

```diff
@@ -44,7 +44,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "health_drain"                                    "150 225 300 500 750"
+        "health_drain"                                    "150 250 350 650 950" //OAA
```

Comment:
With decrepify and spell amp items you think this is a good idea? Meta right now are spell-casters anyway.